services/base.js

```diff
@@ -395,6 +397,7 @@ class BaseService {
           sendBadge(format, badgeData)
         },
         cacheLength: this._cacheLength,
+        fetchLimitBytes: bytes(config.public.fetchLimit),
```

Comment:
This could be passed through `register()` in `lib/server.js`:

https://github.com/badges/shields/blob/afcc0e3920aa7f8f6f80f1ca40b757315f1c02ce/lib/server.js#L181-L185